packages/ui/src/components/TextArea/index.tsx

```diff
@@ -164,7 +168,8 @@ export const TextArea = forwardRef<HTMLTextAreaElement, TextAreaProps>(
       value,
       onChange,
       placeholder,
-      rows = 3,
+      rows = 'auto',
+      autoExpandMax,
```

Comment:
rename in `rowsMax` with a comment to explain it needs to be used in association with `rows="auto"`